warehouse/templates/pages/help.html

```diff
@@ -615,8 +615,25 @@ <h3 id="request-ownership">{{ request_ownership() }}</h3>
         <p>{% trans href='#project-name-claim', anchor_text=project_name_claim() %}If the owner is unresponsive, see <a href="{{ href }}">{{ anchor_text }}</a>{% endtrans %}</p>
 
         <h3 id="description-content-type">{{ description_content_type() }}</h3>
-        <p>{% trans href='https://docutils.sourceforge.io/rst.html', title=gettext('External link') %}By default, an upload's description will render with <a href="{{ href }}" title="{{ title }}" target="_blank" rel="noopener">reStructuredText</a>. If the description is in an alternate format like Markdown, a package may set the <code>long_description_content_type</code> in <code>setup.py</code> to the alternate format.{% endtrans %}</p>
-        <p>{% trans href='https://packaging.python.org/guides/making-a-pypi-friendly-readme/', title=gettext('External link') %}Refer to the <a href="{{ href }}" title="{{ title }}" target="_blank" rel="noopener">Python Packaging User Guide</a> for details on the available formats.{% endtrans %}</p>
+        <p>
+          {% trans %}
+          When using <code>pyproject.toml</code> for project metadata, you can use the suffix of the <code>readme</code> field to control how PyPI renders your description.
+          {% endtrans %}
+        </p>
+        <p>
+          {% trans href='https://docutils.sourceforge.io/rst.html', title=gettext('External link') %}
+          For example, <code>readme = "README.md"</code> will render the description as Markdown,
+          while <code>readme = "README.rst"</code> will render it as
+          <a href="{{ href }}" title="{{ title }}" target="_blank" rel="noopener">reStructuredText</a>
+          {% endtrans %}
+        </p>
+        <p>
+          {% trans href='https://packaging.python.org/guides/making-a-pypi-friendly-readme/', title=gettext('External link') %}
+          Refer to the
+          <a href="{{ href }}" title="{{ title }}" target="_blank" rel="noopener">Python Packaging User Guide</a>
+          for details on the available formats and use with <code>setup.py</code>.
```

Comment:
```suggestion
          for details on the available formats.
```
I think this is fine without this?